vrf-listener/vrf_listener/main.py

```diff
@@ -30,18 +28,13 @@ async def main(
         network=network if rpc_url is None else rpc_url,
         account_contract_address=int(admin_address, 16),
         account_private_key=int(private_key, 16),
-        contract_addresses_config=ContractAddresses(
```

Comment:
The test will now fail because of this removal. Now that we don't pass the `oracle_address`, the main function will fail on its own thread during this:
```python
if not contract_addresses_config:
    contract_addresses_config = CONTRACT_ADDRESSES[self.network]
̀̀```

See:
```
             ^^^^^^^^^^^^^^^^^^^^
  File "/home/akhercha/Programming/pragma/MDR/pragma-sdk/pragma_sdk/onchain/client.py", line 101, in __init__
    contract_addresses_config = CONTRACT_ADDRESSES[self.network]  # type: ignore[index]
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'devnet
```

We have to mock the result of `CONTRACT_ADDRESSES` so it returns the devnet deployed address.